nautobot/extras/tests/test_plugins.py

```diff
@@ -272,6 +282,93 @@ def test_registry_nav_menu_dict(self):
         )
 
 
+@skipIf(
+    "dummy_plugin" not in settings.PLUGINS,
+    "dummy_plugin not in settings.PLUGINS",
+)
+class PluginWebhookTest(SplinterTestCase):
+    """
+    This test case proves that plugins can use the webhook functions when making changes on a model.
+
+    Because webhooks use celery a class variable is set to True called `requires_celery`. This starts
+    a celery instance in a separate thread.
+    """
+
+    requires_celery = True
+
+    def setUp(self):
+        super().setUp()
+        self.url = f"http://localhost:{self.server_thread.port}" + reverse(
+            "plugins-api:dummy_plugin-api:dummymodel_webhook"
+        )
+        self.login(self.user.username, self.password)
+        self.token = Token.objects.create(user=self.user)
+        self.header = f"Authorization: Token {self.token.key}"
+        self.webhook = Webhook.objects.create(
+            name="DummyModel",
+            type_create=True,
+            type_update=True,
+            type_delete=True,
+            payload_url=self.url,
+            http_method=WebhookHttpMethodChoices.METHOD_GET,
+            additional_headers=self.header,
+        )
+        dummy_ct = ContentType.objects.get_for_model(DummyModel)
+        self.webhook.content_types.set([dummy_ct])
```

Comment:
Might be good to explicitly check for and delete any pre-existing `/tmp/test_plugin_webhook_*` files as part of setUp.